connexion/apps/flask_app.py

```diff
@@ -64,6 +65,7 @@ def common_error_handler(self, exception):
     def add_api(self, specification, **kwargs):
         api = super().add_api(specification, **kwargs)
         self.app.register_blueprint(api.blueprint)
+        self.extra_files.append(specification)
```

Comment:
This can be more than just `str`, it can also be a `pathlib.Path` or dict instance

Reply:
Now it will only be added in case it's either string or Path